core/src/main/java/io/ddf/etl/TransformationHandler.java

```diff
@@ -391,4 +379,17 @@ public static String RToSqlUdf(String RExp) {
   @Override public DDF oneHotEncoding(String inputColumn, String outputColumnName) throws DDFException {
     throw new UnsupportedOperationException();
   }
+
+  @Override public DDF castType(String column, String newType) throws DDFException {
+    throw new UnsupportedOperationException();
```

Comment:
Ah... I understand... it's better to make this method abstract rather than method with inPlace flag